ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -625,13 +646,16 @@ static int
 azure_file_size(WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t *sizep)
 {
     azure_file_handle *azure_fh = reinterpret_cast<azure_file_handle *>(file_handle);
+    azure_file_system *azure_fs = azure_fh->fs;
+    auto log = azure_fs->store->log.get();
```

Comment:
I believe we don't need to perform .get(), we can directly just do azure->fs->store->log.

Reply:
azure_log_system is a unique_ptr and in this case we are storing the pointer to the azure_log_system in azure_store in log so the only way to do this without moving ownership is using .get()

Reply:
yep I understand! Sorry to cause confusion but I believe that you can directly still access the the underlying function that the unique pointer is holding. For example

```
auto log = azure_fs->store->log;
log->log_error_message("hi");
```

or 

```
azure_fs->store->log->log_error_message("hi");
```